datamol/io.py

```diff
@@ -317,7 +316,7 @@ def read_smi(
     # Copy to a local temporary path if the path is a remote one.
     if not fsspec.utils.can_be_local(str(urlpath)):
         active_path = pathlib.Path(tempfile.mkstemp()[1])
-        dm.utils.fs.copy_file(urlpath, active_path)
+        dm.utils.fs.copy_file(urlpath, active_path, force=True)
```

Comment:
@hadim, tempfile has  `tempfile._get_candidate_names()` to generate the name without creating the file first.

Reply:
Nice, I didn't know about that.

That being said, I prefer keeping it like that since `_get_candidate_names` seems to be a private method and so is not guaranteed by the Python API (from my experience using private RDKit methods).